x/config/chooser.go

```diff
@@ -68,62 +68,62 @@ import (
 //
 // Integration
 //
-// To integrate peer lists with your transport, embed this struct into your
+// To integrate peer choosers with your transport, embed this struct into your
 // outbound configuration.
 //
 // 	type myOutboundConfig struct {
-// 		config.PeerList
+// 		config.PeerChooser
 // 		Address string
 // 	}
 //
-// Then in your Build*Outbound function, use the PeerList.BuildPeerList method
+// Then in your Build*Outbound function, use the PeerChooser.Build method
 // to retrieve a peer chooser for your outbound. The following example only
 // works if your transport implements the peer.Transport interface.
 //
 // 	func buildOutbound(cfg *myOutboundConfig, t transport.Transport, k *config.Kit) (transport.UnaryOutbound, error) {
 // 		myTransport := t.(*MyTransport)
-// 		peerChooser, err := cfg.PeerList.BuildPeerList(myTransport, hostport.Identify, k)
+// 		peerChooser, err := cfg.PeerChooser.BuildPeerChooser(myTransport, hostport.Identify, k)
 // 		if err != nil {
 // 			return nil, err
 // 		}
 // 		return myTransport.NewOutbound(peerChooser), nil
 // 	}
 //
 // The *config.Kit received by the Build*Outbound function MUST be passed to
-// the BuildPeerList as-is.
+// the Build function as-is.
 //
-// Note that the keys for the PeerList share the namespace with the attributes
+// Note that the keys for the PeerChooser share the namespace with the attributes
 // of your outbound configuration.
```

Comment:
We should enumerate those keys in the docs, since the default GoDoc view won't show the private `peerChooser` struct below.